pandasdmx/api.py

```diff
@@ -388,7 +388,7 @@ def get(self, resource_type=None, resource_id=None, tofile=None,
                 raise
 
         # Maybe copy the response to file as it's received
-        arg = [tofile] if tofile else []
+        arg = [tofile] if tofile else [] # explain or remove this. Why wrap in list and unpack it in next line?
```

Comment:
This is a common pattern in Python argument handling.

If 'tofile' is `None` (by default, or explicitly) it makes arg an empty list, i.e. the following call is equivalent to `remote.ResponseIO(response)`.